File: FullStack_Developer/1_java/workspace/day02/src/day02/WhileTest.java
package day02;

import java.util.Scanner;

public class WhileTest {
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		
		//1. while 문
//		String choice = "";
//		while(!choice.equals("0")) {
//			System.out.println("다음 중 프로그래밍 언어가 아닌것은?\n"
//					+ "1. C언어\n2. Java\n3. 망둥어\n4. 파이썬\n0. 나가기");
//			choice = sc.next();
//			
//			if(choice.equals("3")) {
//				System.out.println("정답입니다.");
//			}
//			else if(choice.equals("1") || choice.equals("2") || choice.equals("4")) {
//				System.out.println("오답입니다.");
//			}
//			else if(choice.equals("0")) {
//				System.out.println("안녕히ㄱr세요...☆");
//			}
//			else {
//				System.out.println("잘못 입력하셨습니다.");
//			}
//		}
		
		//2. do ~ while 문
//		String choice = "0";
//		do{
//			System.out.println("다음 중 프로그래밍 언어가 아닌것은?\n"
//					+ "1. C언어\n2. Java\n3. 망둥어\n4. 파이썬\n0. 나가기");
//			choice = sc.next();
//			
//			if(choice.equals("3")) {
//				System.out.println("정답입니다.");
//			}
//			else if(choice.equals("1") || choice.equals("2") || choice.equals("4")) {
//				System.out.println("오답입니다.");
//			}
//			else if(choice.equals("0")) {
//				System.out.println("안녕히ㄱr세요...☆");
//			}
//			else {
//				System.out.println("잘못 입력하셨습니다.");
//			}
//		}while(!choice.equals("0"));
		
		//3. 무한반복
		String choice = "0";
		while(true){
			System.out.println("다음 중 프로그래밍 언어가 아닌것은?\n"
					+ "1. C언어\n2. Java\n3. 망둥어\n4. 파이썬\n0. 나가기");
			choice = sc.next();
			
			if(choice.equals("3")) {
				System.out.println("정답입니다.");
			}
			else if(choice.equals("1") || choice.equals("2") || choice.equals("4")) {
				System.out.println("오답입니다.");
			}
			else if(choice.equals("0")) {
				System.out.println("안녕히ㄱr세요...☆");
				break;
			}
			else {
				System.out.println("잘못 입력하셨습니다.");
			}
		}
	}
}

File: Basic_Language/2_Python/workspace/day03/WhileTest.py
#WhileTest.py
choice = '0'
'''
while choice != "3":
    print("다음 중 프로그래밍 언어가 아닌것은?\n1. 파이썬\n2. C언어\n3. 망둥어\n4. JAVA")
    choice = input()
    if choice == "3":
        print("정답입니다")
    elif choice == "1" or choice == "2" or choice == "4":
        print("오답입니다.")
    else:
        print("잘못 입력하셨습니다.")
'''
while True:
    print("다음 중 프로그래밍 언어가 아닌것은?\n\
1. 파이썬\n2. C언어\n3. 망둥어\n4. JAVA\n0. 나가기")
    choice = input()
    if choice == "3":
        print("정답입니다")
    elif choice == "1" or choice == "2" or choice == "4":
        print("오답입니다.")
    elif choice == "0":
        print("안녕히가세요")
        break
    else:
        print("잘못 입력하셨습니다.")




















        
